provider/pprof/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@openctx/provider-pprof",
+  "version": "0.0.13",
+  "description": "pprof (OpenCtx provider)",
+  "license": "Apache-2.0",
+  "homepage": "https://openctx.org/docs/providers/pprof",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcegraph/openctx",
+    "directory": "provider/pprof"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": ["dist/index.js", "dist/index.d.ts"],
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --build",
+    "bundle": "tsc --build && esbuild --log-level=error --bundle --format=cjs --platform=node --external:vscode --outfile=dist/bundle.js index.ts",
```

Comment:
Interesting, does this work for you with `--format=cjs` ? (I'm trying to do something similar but from what I can tell the `importHelper.ts` always assumes it's esm.

Reply:
It does, although it's a temporary patch until [this fix](https://github.com/sourcegraph/openctx/pull/131) is released.

You're right, `importHelper.ts` assumes the bundles are `esm`, but will still try to convert them to `cjs` if it's run on desktop. Since this bundle is already in `cjs` format it doesn't break things.

Reply:
Updated to `--format=esm`, the latest OpenCtx version can now compile it correctly.